app/main.py

```diff
@@ -28,6 +28,21 @@
 )
 
 
+@app.get("/", response_class=HTMLResponse)
+def root():
+    """
+    Display a welcome message and a link to the API documentation.
+    """
+    return """
+    <html>
+        <body>
+            <h1>Welcome to the Neurobagel REST API!</h1>
+            <p>Please visit the <a href="/docs">documentation</a> to view available API endpoints.</p>
```

Comment:
> If /docs part could be made a link to the <base-url>/docs, that would also be cool but is not a necessity

@alyssadai was this comment from #199 in reference to running the API behind a proxy?

Reply:
Nope, I was thinking of a hyperlink that takes you to the API docs, to make it easier for a user to know where to go next when they hit this welcome page (most useful when no NGINX redirects have been set up). So this looks fine to me!